### PR TITLE
Ingesting EAMT archives

### PR DESCRIPTION
**Pull request by mjpost**
I'm ready to push this through now. I'm going to add one or two a day until they're ready. Supplants #965 #966.

I am tracking the status [here](https://docs.google.com/spreadsheets/d/1fpxmdV_BPwR6BQHyU9VJQxXeSOmy4__5nQCHBEviyAw/edit#gid=509047314) as I ingest these.

Status:
- [x] 2008
- [x] 2009
- [x] 2010
- [x] 2011
- [x] 2012
- [x] 2013 (none, apparently)
- [x] 2014
- [x] 2015
- [x] 2016 (already in Anthology, but user track needs splitting out)
- <strike>[ ] 2017 (??)</strike>
- <strike>[ ] 2018 (??)</strike>
- <strike>[ ] 2019 (??)</strike>

### CODE REVIEW

File: data/xml/2005.eamt.xml [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+    <paper id="37">
+      <title>Sentence segmentation using <fixed-case>IBM</fixed-case> word alignment model 1</title>
+      <author><first>Jia</first><last>Xu</last></author>
+      <author><first>Richard</first><last>Zens</last></author>
+      <author><first>Hermann</first><last>Ney</last></author>
+      <url hash="2951fc50">2005.eamt-1.37</url>
+    </paper>
+    <paper id="38">
+      <title>Initial considerations in building a speech-to-speech translation system for the <fixed-case>S</fixed-case>lovenian-<fixed-case>E</fixed-case>nglish language pair</title>
+      <author><first>J.</first><last>ŽganecGros</last></author>
```
Comment (danielgildea) [inline data/xml/2005.eamt.xml:273]: space in Žganec Gros

File: data/xml/2005.eamt.xml [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+      <author><first>A.</first><last>Mihelič</last></author>
+      <author><first>M.</first><last>Žganec</last></author>
+      <author><first>F.</first><last>Mihelič</last></author>
+      <author><first>S.</first><last>Dobrišek</last></author>
+      <author><first>J.</first><last>Žibert</last></author>
+      <author><first>Š.</first><last>Vintar</last></author>
+      <author><first>T.</first><last>Korošec</last></author>
+      <author><first>T.</first><last>Erjavec</last></author>
+      <author><first></first><last>Romih,</last></author>
+      <author><first></first><last>M.</last></author>
```
Comment (danielgildea) [inline data/xml/2005.eamt.xml:283]: one person: M. Romih

File: data/xml/1993.eamt.xml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    <meta>
+      <booktitle>Third International EAMT Workshop: Machine Translation and the Lexicon</booktitle>
+      <publisher>Springer</publisher>
+      <address>Heidelberg, Germany</address>
+      <month>April 26–28</month>
+      <year>1993</year>
+      <editor><first>Petra</first><last>Steffens</last></editor>
+      <isbn>3-540-59040-4</isbn>
+    </meta>
+    <paper id="1">
```
Comment (danielgildea) [inline data/xml/1993.eamt.xml:14]: Add introduction? would be consistent with other years.
https://link.springer.com/book/10.1007/3-540-59040-4

File: data/xml/2005.eamt.xml (hunk 1 above)
Comment (mjpost) [reply data/xml/2005.eamt.xml]: Done

File: data/xml/2005.eamt.xml (hunk 2 above)
Comment (mjpost) [reply data/xml/2005.eamt.xml]: Done.

File: data/xml/1993.eamt.xml (hunk 3 above)
Comment (mjpost) [reply data/xml/1993.eamt.xml:13]: Ah, good catch. Thank you for your careful reviews!

Comment (mjpost) [reply data/xml/1993.eamt.xml:13]: Comparing that link to that of the MT Archive, it seems we are also missing the paper [Machine Translation and the Lexicon](https://link.springer.com/book/10.1007/3-540-59040-4). I'll add it.

**Comment by mjpost** [comment]
For EAMT 2011, I included the project descriptions ([at the bottom of this page](http://www.mt-archive.info/10/EAMT-2011-TOC.htm)). I'm not sure if that was the right choice, but it doesn't seem to hurt.

**Comment by mjpost** [comment]
Hi @danielgildea—this should now be completed, as I have moved over everything from the MT Archive. If you have a chance to look it over today or tomorrow, EAMT would be grateful (Mikel is hoping to announce on Weds).

File: data/xml/2011.eamt.xml [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+      <title>Exploiting Objective Annotations for Minimising Translation Post-editing Effort</title>
+      <author><first>Lucia</first><last>Specia</last></author>
+      <url hash="fe639d89">2011.eamt-1.12</url>
+      <attachment type="presentation" hash="7dfa5ab3">2011.eamt-1.12.Presentation.pdf</attachment>
+    </paper>
+    <paper id="13">
+      <title>Using word alignments to assist computer-aided translation users by marking which target-side words to change or keep unedited</title>
+      <author><first>Miquel</first><last>Esplà</last></author>
+      <author><first>Felipe</first><last>Sánchez-Martínez</last></author>
+      <author><first>Mikel L</first><last>Forcada</last></author>
```
Comment (danielgildea) [inline data/xml/2011.eamt.xml:108]: L.

**Review by danielgildea — APPROVED** [review]

File: data/xml/2011.eamt.xml (hunk 4 above)
Comment (mjpost) [reply data/xml/2011.eamt.xml]: Fixed.

**Comment by mjpost** [comment]
Awesome, thank you! I may sneak in a few UI fixes, and then will spend some time consolidating the merge commit message later today.

**Comment by danielgildea** [comment]
I just noticed that EAMT 2015 was already present in W15.xml - now it appears twice.

**Comment by mjpost** [comment]
It's actually two volumes—the second is user papers. It used to appear as a single "paper" in the first volume, but I split it out. I know I updated the metadata for the second volume, but I must have overwritten it. I'll push it up on emnlp-fixes.

**Comment by mjpost** [comment]
I see. It's actually all one volume originally. So it's too bad it's displayed twice, but I think the volume titles are actually appropriate.

**Pull request merged**